rdjson_formatter/rdjson_formatter.rb

```diff
@@ -90,23 +90,44 @@ def build_range(offense)
   # @param [RuboCop::Cop::Offense] offense
   # @return [Array{Hash}]
   def build_suggestions(offense)
```

Comment:
This has two Rubocop metrics based warnings since it's a larger method now. Perhaps determining the positions/range and sorted corrections could each be extracted into a private method?

We don't see to have tests for this other than integration tests through Rubocop CLI. If its easier to write a unit test that would be fine but is there anyway we could update the test data with samples that would trigger the warnings before this change and then include it here to demonstrate that the fix is working as expected?